doc/docs/master/assets/images/d_steps_build_pipeline.svg

```diff
@@ -0,0 +1 @@
5 1.09375 0.390625q0.34375 0 0.671875 -0.109375q0.328125 -0.125 0.5625 -0.328125l0.40625 0.8125zm21.875519 -10.6875l-3.78125 11.203125l-1.25 0l-3.28125 -9.546875l-3.296875 9.546875l-1.21875 0l-3.78125 -11.203125l1.21875 0l3.234375 9.640625l3.34375 -9.640625l1.09375 0l3.296875 9.6875l3.28125 -9.6875l1.140625 0zm4.7241516 11.28125q-1.21875 0 -2.203125 -0.546875q-0.96875 -0.5625 -1.53125 -1.53125q-0.5625 -0.96875 -0.5625 -2.203125q0 -1.234375 0.5625 -2.203125q0.5625 -0.984375 1.53125 -1.53125q0.984375 -0.546875 2.203125 -0.546875q1.203125 0 2.171875 0.546875q0.984375 0.546875 1.53125 1.53125q0.5625 0.96875 0.5625 2.203125q0 1.234375 -0.5625 2.203125q-0.546875 0.96875 -1.53125 1.53125q-0.96875 0.546875 -2.171875 0.546875zm0 -1.0q0.890625 0 1.59375 -0.40625q0.71875 -0.421875 1.109375 -1.171875q0.40625 -0.75 0.40625 -1.703125q0 -0.96875 -0.40625 -1.71875q-0.390625 -0.75 -1.109375 -1.15625q-0.703125 -0.40625 -1.59375 -0.40625q-0.90625 0 -1.625 0.40625q-0.703125 0.40625 -1.109375 1.15625q-0.40625 0.75 -0.40625 1.71875q0 0.953125 0.40625 1.703125q0.40625 0.75 1.109375 1.171875q0.71875 0.40625 1.625 0.40625zm7.672638 -5.84375q0.40625 -0.84375 1.1875 -1.28125q0.796875 -0.4375 1.96875 -0.4375l0 1.109375l-0.28125 -0.015625q-1.328125 0 -2.078125 0.8125q-0.75 0.8125 -0.75 2.296875l0 4.28125l-1.140625 0l0 -8.421875l1.09375 0l0 1.65625zm8.319275 2.546875l-1.984375 1.8125l0 2.40625l-1.140625 0l0 -11.875l1.140625 0l0 8.046875l5.03125 -4.59375l1.40625 0l-3.609375 3.453125l3.9375 4.96875l-1.390625 0l-3.390625 -4.21875zm9.287872 -6.765625q-0.75 0 -1.125 0.40625q-0.375 0.390625 -0.375 1.171875l0 0.984375l2.59375 0l0 0.96875l-2.5625 0l0 7.453125l-1.125 0l0 -7.453125l-1.515625 0l0 -0.96875l1.515625 0l0 -1.03125q0 -1.15625 0.65625 -1.828125q0.671875 -0.671875 1.875 -0.671875q0.484375 0 0.9375 0.140625q0.453125 0.125 0.75 0.390625l-0.390625 0.84375q-0.484375 -0.40625 -1.234375 -0.40625zm2.625 -0.890625l1.140625 0l0 11.875l-1.140625 0l0 -11.875zm7.7592773 11.953125q-1.21875 0 -2.203125 -0.546875q-0.96875 -0.5625 -1.53125 -1.53125q-0.5625 -0.96875 -0.5625 -2.203125q0 -1.234375 0.5625 -2.203125q0.5625 -0.984375 1.53125 -1.53125q0.984375 -0.546875 2.203125 -0.546875q1.203125 0 2.171875 0.546875q0.984375 0.546875 1.53125 1.53125q0.5625 0.96875 0.5625 2.203125q0 1.234375 -0.5625 2.203125q-0.546875 0.96875 -1.53125 1.53125q-0.96875 0.546875 -2.171875 0.546875zm0 -1.0q0.890625 0 1.59375 -0.40625q0.71875 -0.421875 1.109375 -1.171875q0.40625 -0.75 0.40625 -1.703125q0 -0.96875 -0.40625 -1.71875q-0.390625 -0.75 -1.109375 -1.15625q-0.703125 -0.40625 -1.59375 -0.40625q-0.90625 0 -1.625 0.40625q-0.703125 0.40625 -1.109375 1.15625q-0.40625 0.75 -0.40625 1.71875q0 0.953125 0.40625 1.703125q0.40625 0.75 1.109375 1.171875q0.71875 0.40625 1.625 0.40625zm18.698273 -7.5l-3.1875 8.421875l-1.0625 0l-2.65625 -6.890625l-2.65625 6.890625l-1.078125 0l-3.171875 -8.421875l1.09375 0l2.640625 7.15625l2.703125 -7.15625l0.96875 0l2.6875 7.15625l2.671875 -7.15625l1.046875 0z" fill-rule="nonzero"/></g></svg>
```

Comment:
I think all the arrow should be heavier weight like the ones that show the build pipeline process are. Maybe I'm wrong, I'd have to see it I think, but i nearly lose sight of the fact that the actual pipeline still just take input data as well.

Reply:
it might be as simple as just making the arrow heads bigger or weighted more.

Reply:
This comment applies to the other image too.